src/core/electric_utility.jl

```diff
@@ -29,35 +29,56 @@
 # *********************************************************************************
 """
     ElectricUtility
-
 ElectricUtility data struct with inner constructor:     
+        
 ```julia
 function ElectricUtility(;
-    outage_start_time_step::Int=0,  # for modeling a single outage, with critical load spliced into the baseline load ...
-    outage_end_time_step::Int=0,  # ... utiltity production_factor = 0 during the outage
-    allow_simultaneous_export_import::Bool = true,  # if true the site has two meters (in effect)
-    # variables below used for minimax the expected outage cost,
+    emissions_region::String = "",
```

Comment:
Should the user be able to provide an emissions region, as opposed to only being able to use the `region()` function to obtain this?

Reply:
I don't think so. The user provides lat/long, which determined the emissions region.

Reply:
I didn't know this. I guess we do need to add emissions_region as a site input then.

Reply:
Amanda and I talked, and the API doesn't provide this option, so I won't include it in the Julia package now either.